spec/models/form_submission_spec.rb

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require 'securerandom'
```

Comment:
Do we need this line? I feel like I've used `SecureRandom` without it before in this codebase but I could be totally wrong.

Reply:
I'll try removing it.